lib/LanguageServerCompletion/Handler/CompletionHandler.php

```diff
@@ -100,29 +109,42 @@ public function completion(CompletionParams $params, CancellationToken $token):
             $items = [];
             $isIncomplete = false;
             foreach ($suggestions as $suggestion) {
+                /** @var Suggestion $suggestion */
```

Comment:
typically using `assert($suggestion instanceof Suggestion` here. It helps with refactoring.

Reply:
Done